SimPEG/Examples/MT_1D_analytic_nlayer_Earth.py

```diff
@@ -0,0 +1,428 @@
+from scipy.constants import epsilon_0, mu_0
```

Comment:
Due to the complexity of this example, I think it is better suited for the em_examples repository (https://github.com/ubcgif/em_examples/). I believe it is already included there (@thast, can you confirm?). If so, then this script should be removed from here prior to merging the pr